libc-bottom-half/headers/public/wasi/api.h

```diff
@@ -40,16 +41,22 @@ extern "C" {
 #define __WASI_DIRCOOKIE_START (UINT64_C(0))
 typedef __SIZE_TYPE__ __wasi_size_t;
 
+_Static_assert(sizeof(__wasi_size_t) == 4, "witx calculated size");
+_Static_assert(_Alignof(__wasi_size_t) == 4, "witx calculated align");
```

Comment:
Purely for tidiness reasons, it'd be nice to have a newline between these asserts and the subsequent comment.

Reply:
Fixed

Reply:
Fixed in 475a62